### PR TITLE
Fix copy-to-clipboard in Firefox.

### LINKED ISSUE #848: Copy/paste in docs has issues in Firefox and IE
I've just visted the link [Material Design Lite - buttons](http://www.getmdl.io/components/index.html#buttons-section)
~~I'm so suprised to find that the ripple animation is using a swf file since I'm using the latest version of chrome on a macbook, and the animation is too fast to be noticed.~~
~~CSS animations are supported in latest browsers (including IE10) and google chrome are planning to drop the flash support (Apple's Safari already does this), I think it would be better if use css animation instead of flash. :(~~
thanks to @mkhairi @steven-qi 's mention, the flash file is for coping the code snippet. It's a bad idea to use adobe flash in a website like this.

### PR DESCRIPTION
**Pull request by nicolasgarnier**
r: @addyosmani 
PS: I havn't tested this in IE. Still need to install VMs :(

Closes #848

### CODE REVIEW

**Comment by nicolasgarnier** [comment]
PS: Flash seems to have an issue in Firefox where not all click-related events are passed to the parent elements of the flash movie (depsite `wmode: transparent`) so the ripple effect on the button would be broken: The ripple started but it would stay mousedown position. So I simply removed the ripple from the buttons.

**Comment by nicolasgarnier** [comment]
Closes #848

**Comment by surma** [comment]
screenshot 2015-07-08 14 32 59

**Comment by surma** [comment]
I just rebuild the repo, the error disappeared, but Copy/Paste still does not work.

**Comment by nicolasgarnier** [comment]
Surma can you try again now?

**Comment by nicolasgarnier** [comment]
FYI @surma  if you don't get the "copy" icon this means that you have flash disabled on your browser :)

**Comment by nicolasgarnier** [comment]
@addyosmani PTAL

**Comment by surma** [comment]
Works for me :smirk:

**Comment by surma** [comment]
IE is not working, though.

**Comment by nicolasgarnier** [comment]
@surma is trying again after installing flash ;) There is hope :D

**Comment by surma** [comment]
I said it before, and I will say it again: IE totally works :sweat_smile:

**Comment by nicolasgarnier** [comment]
\O/

**Comment by nicolasgarnier** [comment]
Done PTAL!

File: docs/_assets/snippets.js [hunk 1]
```diff
@@ -27,15 +28,23 @@ function MaterialComponentsSnippets() {
  */
 MaterialComponentsSnippets.prototype.init = function() {
```
Comment (samccone) [inline docs/_assets/snippets.js]: will this ever be inited outside of window.onload? 
If so should we think about providing a destroy method to clean these listeners?

Comment (nicolasgarnier) [inline docs/_assets/snippets.js]: afaik this should be fine. This is just for the documentation website and for now will only ever be used inside `window.onload()`

Comment (samccone) [inline docs/_assets/snippets.js]: kk :)

**Comment by addyosmani** [comment]
Verified that this now works in FF stable, Mac. Testing IE next.

**Comment by addyosmani** [comment]
Tested in IE11 and works fine there. I'll verify in IE10 and then merge.

**Comment by addyosmani** [comment]
Tested in IE10 works there too. Landed this via merge commit in b9b20c41e0a5d47f6687aa6f9ddb7c4c569dcbd3

**Comment by addyosmani** [comment]
Nice work, @nicolasgarnier!

**Pull request closed without merging**